app/app.go

```diff
@@ -290,6 +297,8 @@ type App struct {
 	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
 	ScopedWasmKeeper     capabilitykeeper.ScopedKeeper
 
+	AccessControlKeeper aclmodulekeeper.Keeper
```

Comment:
Since we're introducing a new keeper, we likely need to add a migration handler for the store too: https://github.com/sei-protocol/sei-chain/blob/master/app/app.go#L750